source/renderer/app/config/timingConfig.js

```diff
@@ -11,3 +11,4 @@ export const MAX_ALLOWED_STALL_DURATION = 2 * 60 * 1000; // 2 minutes | unit: mi
 export const NETWORK_STATUS_REQUEST_TIMEOUT = 30 * 1000; // 30 seconds | unit: milliseconds
 export const NETWORK_STATUS_POLL_INTERVAL = 2000; // 2 seconds | unit: milliseconds
 export const NTP_FORCE_CHECK_POLL_INTERVAL = 30 * 60 * 1000; // 30 minutes | unit: milliseconds
+export const NTP_IGNORE_CHECKS_GRACE_PERIOD = 10 * 1000; // 30 seconds | unit: milliseconds
```

Comment:
@DominikGuzei this doesn't look like 30 seconds 🤔 😄

Reply:
Ah thanks, forgot to change it back 👍  that was just an OCD test and you passed! 😋